lib/item_repository.rb

```diff
@@ -35,7 +35,7 @@ def update(id, attributes)
       updated_at: Time.now
     }.compact
     super(id, sanitized_attributes)
-  end 
+  end
 
   def inspect
```

Comment:
Can we remove these individual def inspect methods from all but parent repository and retest for pass?

Reply:
Done! and retested all 🟢